curriculum/challenges/english/25-front-end-development/workshop-merge-sort/656702f8b4cbd8cbf0a433c6.md

```diff
@@ -0,0 +1,54 @@
+---
+id: 656702f8b4cbd8cbf0a433c6
+title: Step 11
+challengeType: 20
+dashedName: step-11
+---
+
+# --description--
+
+Inside your function, create a `while` loop that compares an element in `left_part` to an element in `right_part`, and merges the smaller element to the main `array` list. 
```

Comment:
This is misleading. We need to explain the purpose of the loop, but said like this it seems you need to do it in this step.